### DISCUSSION TITLE
Technology news

### DISCUSSION THREAD

**Discussion by nappalm** [dailydotdev/daily#584] [Feature Requests]
Please add technology news, to know what happens every day in the technology industry.

**Comment by idoshamun** [comment]
Hi @nappalm,

We have several technology news sources like Tech Crunch and The Next Web.
Can't you see them in your feed?

**Reply by nappalm** [reply]
Yes I see it, but I really think there is a lack of better sources for this section

**Reply by idoshamun** [reply]
Yap. It's not our primary focus at the moment. We look for engineering/development oriented content primarily at the moment

**Comment by rebelchris** [comment]
@nappalm You can suggest new sources via the app.
In these docs we explain how you can do so, we would love to see some more quality sources 💖

https://docs.daily.dev/docs/for-content-creators/suggest-new-source
